fetcher/src/job/job.reducer.ts

```diff
@@ -41,6 +42,13 @@ export function mulFn(args: number) {
   return wrapper
 }
 
+export function divFromFn(args: number) {
+  function wrapper(value: number) {
```

Comment:
I am not sure if it can make any problem, but there is has a chance when `value` is zero it might make some problem. 

Reference: https://www.educative.io/answers/how-does-javascript-handle-divide-by-zero

Reply:
That is good point! @nick-bisonai Please suggest a solution to it 🙏

Reply:
@martinkersner , @bayram98 
I'll create a new `FetcherError` which stands for division by zero error and throw in case that happens.